.dotnet.azure/sdk/openai/Azure.AI.OpenAI/tests/Samples/01_Chat.cs

```diff
@@ -255,7 +255,7 @@ in chatClient.CompleteChatStreaming(conversationMessages, options))
                 {
                     functionNamesByIndex[toolCallUpdate.Index] = toolCallUpdate.FunctionName;
                 }
-                if (!string.IsNullOrEmpty(toolCallUpdate.FunctionArgumentsUpdate))
+                if (!string.IsNullOrEmpty(toolCallUpdate.FunctionArgumentsUpdate.ToString()))
```

Comment:
same, can you do Lenght == 0

Reply:
I have now implemented the builder in this AOAI sample too.